src/test/java/co/rsk/federate/btcreleaseclient/BtcReleaseClientTest.java

```diff
@@ -439,16 +495,56 @@ public void onBlock_return_when_node_is_syncing() {
         );
         btcReleaseClient.start(federation);
 
+        List<TransactionReceipt> receipts = new ArrayList<>();
+        TransactionReceipt transactionReceipt = mock(TransactionReceipt.class);
+        receipts.add(transactionReceipt);
+
+        // Act
+        ethereumListener.get().onBlock(null, receipts);
+
+        // Assert
+        verifyZeroInteractions(transactionReceipt);
```

Comment:
Let's add one more assertion:

```suggestion
        verify(nodeBlockProcessor, times(1)).hasBetterBlockToSync();
        verifyZeroInteractions(transactionReceipt);
```